tests/Caching/SQLiteJournal.phpt

```diff
@@ -21,7 +21,7 @@ class SQLiteJournalTest extends IJournalTestCase
 
 	public function createJournal()
 	{
-		return new SQLiteJournal;
+		return new SQLiteJournal(TEMP_DIR . '/sqlitejournal_' . uniqid() . '.sqlite');
```

Comment:
For "in memory" SQLite storage there were problems :o 

See https://travis-ci.org/nette/caching/builds/71373010


It looks like that test cases shared the same database but it was OK before my changes, I don't know why...


---

Now, it is OK: https://travis-ci.org/nette/caching/builds/71373257